pythran/optimizations/constant_folding.py

```diff
@@ -7,7 +7,6 @@
 from pythran.analyses.ast_matcher import DamnTooLongPattern
 
 import ast
-import numpy
```

Comment:
On Wed, Sep 02, 2015 at 11:39:51AM -0700, pbrunet wrote:
> I think numpy is required here. It is not in the code but eval may need to load it. (If I correctly remember... sorry for not documenting this ...)

I would say that it's already handled in the prepare method.
At least running the test_cases.py validation does not yield any
error...